keras_cv/utils/rle_utils.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import tensorflow as tf
+
+
+def rle_to_mask2d(mask_rle: str, shape) -> tf.Tensor:
+    """Converts a Run-length encoded segmentation map to a 2-dimensional segmentation map.
+
+    Args:
+      mask_rle: a string that represents a run-length encoded segmentation map.
+      shape: shape of the resultant 2-dimensional segmentation map.
+
+    Returns:
+      mask: 2-dimensional segmentation map.
+    """
```

Comment:
Include:

* A `Raises` flag (example) that discusses a corner case if that's known. 
* A sample example.